src/Components/SmartComponents/Modals/CvePairStatusModal.js

```diff
@@ -70,13 +70,17 @@ export const CvePairStatusModal = ({ cves, updateRef, inventories, hasDifferentS
     }
 
     function getDefaultCheckboxState() {
+
+        const isSameCves = cveList.every(
```

Comment:
I'd probably rename this to `haveCvesSameStatus` because `isSameCves` is a bit misleading.